shippy-lib/shippy-trace.js

```diff
@@ -0,0 +1,52 @@
+let Trace = (function () {
```

Comment:
This is a JS global on the `window` object. It would be nicer to have it on our global `Shippy` object as `Shippy.Trace`. But there may be issues if you include the file in index.html after shippy.js and then use the exposed methods in shippy.js before DOMReady. But that can be fixed too...